extensions/amp-story/1.0/amp-story.css

```diff
@@ -273,6 +273,22 @@ amp-story-grid-layer > * {
   pointer-events: auto !important;
 }
 
+.i-amphtml-story-grid-layer-z-index-reset amp-story-grid-layer,
+.i-amphtml-story-grid-layer-z-index-reset amp-story-grid-layer * {
+  contain: none !important;
+  z-index: auto !important;
+}
+
+amp-story-page.i-amphtml-story-grid-layer-z-index-reset[active] {
+  contain: none !important;
+  z-index: 100001 !important; /* On top of i-amphtml-story-system-layer. */
+}
+
+amp-twitter.i-amphtml-increment-z-index {
+  z-index: 2 !important;
+  position: relative !important;
```

Comment:
This might break the publisher's layout and break your animation. eg: the position was absolute, and switching to relative changes the element's position.

Reply:
Done.